### PR TITLE
W-8950994 - OFM: 230 sandbox push upgrade failures - remove flexipage report references

### PR DESCRIPTION
**Pull request by seanpat09**
Currently managed reports can be deleted even if they are currently referenced in managed flexipages. If the customer does this, then push upgrades will fail due to this missing metadata. To mitigate this from happening, this PR removes references to reports in the Funding_Program_Record_Page flexipage.

This flexipage previously had a tab called "Program Analysis" that would display one of two reports based one whether the field `Top Level` was set to true on the Funding Program. If `Top Level` is set to `true`, the tab would also contain a related list for Subordinate Funding Programs. See screenshot for previous behavior:

Screen Shot 2021-03-03 at 4 46 44 PM

In this PR, that "Program Analysis" tab is completely removed. If `Top Level` is set to `true`, the Subordinate Funding Programs related list now appears on the detail page. See screenshot for new behavior:

Screen Shot 2021-03-03 at 4 38 27 PM

# Critical Changes

- To prevent the Outbound Funds Module installer from failing from failing to upgrade your org, we removed the Program Analysis tab from the Funding Program page. If you want the Program Analysis tab to display, you can add it back after your org is upgraded.
1. Navigate to a Funding Program.
2. Click **Setup**, then click **Edit Page**.
3. Click the tab headings section.
4. Click **Add Tab**.
5. Click the new tab, then select **Program Analysis**.
6. Click **Save**.
# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done
Refer to [Definition of Done](https://salesforce.quip.com/9P7hAOPHJJyU) to see any additional details for the items below:
- [ ] ~Any net new LWC work has JEST test coverage 50% or above~
- [ ] ~Default Sa11y tests pass for all LWC components~
- [ ] ~🔒 Secure both Front-end (LWC) & back-end (Apex) as necessary~
- [ ] ~🔑 Grant users access in Permission Sets (Object, Field, Apex Class) as necessary~
- [x] Link the pull request and work item by PR comment and Chatter post respectively, e.g. GUS: [W-0000000: Work Name]()
- [x] Make sure that ACs are updated (if any gaps)
- [x] **All acceptance criteria have been met**
    - [x] Developer
    - [x] Code Reviewer
- [ ] Pull Request contains draft release notes
- [ ] Labels, help text, and customer facing messages are reviewed by Docs
- [x] QE story level testing completed

# Unaggregated Pull Requests
* Update admin profile + support 2GP [[PR93](https://github.com/SalesforceFoundation/OutboundFunds/pull/93)]

### CODE REVIEW

**Comment by seanpat09** [comment]
[W-8950994 - OFM: 230 sandbox push upgrade failures - remove flexipage report references](https://gus.lightning.force.com/a07B00000097lhkIAA)

File: force-app/main/default/flexipages/Funding_Program_Record_Page.flexipage-meta.xml [hunk 1]
```diff
@@ -6,6 +6,14 @@
...
                 <value>false</value>
             </componentInstanceProperties>
+            <componentInstanceProperties>
+                <name>enableActionsConfiguration</name>
+                <value>false</value>
+            </componentInstanceProperties>
+            <componentInstanceProperties>
+                <name>hideChatterActions</name>
+                <value>false</value>
+            </componentInstanceProperties>
```
Comment (spelak-salesforce) [inline force-app/main/default/flexipages/Funding_Program_Record_Page.flexipage-meta.xml:16]: Must be new default FlexiPage config?

**Review by spelak-salesforce — APPROVED** [review]
Great fix @seanpat09 removing the report chart + FlexiPage section since the section is empty without the report chart.

File: lib/admin.profile-meta.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jstvz) [inline lib/admin.profile-meta.xml:1]: Default XML:
```xml
<Package xmlns="http://soap.sforce.com/2006/04/metadata">
    <types>
        <members>*</members>
        <members>Account</members>
        <members>Campaign</members>
        <members>CampaignMember</members>
        <members>Contact</members>
        <members>Lead</members>
        <members>Opportunity</members>
        <name>CustomObject</name>
    </types>
    <types>
        <name>Profile</name>
    </types>
    <version>39.0</version>
</Package>
```

File: force-app/main/default/flexipages/Funding_Program_Record_Page.flexipage-meta.xml (hunk 1 above)
Comment (seanpat09) [reply force-app/main/default/flexipages/Funding_Program_Record_Page.flexipage-meta.xml:16]: Yup, these just got added when I pulled down the org changes

**Review by jstvz — APPROVED** [review]
`CODEOWNERS` 🆗

File: lib/admin.profile-meta.xml (hunk 2 above)
Comment (seanpat09) [reply lib/admin.profile-meta.xml:1]: @jstvz Should I update this file to include the Standard objects you have listed in here?

Comment (jstvz) [reply lib/admin.profile-meta.xml:1]: Any of the missing standard objects you want to keep will need to be added here.

**Pull request merged**